pkg/arvo/app/contact-push-hook.hoon

```diff
@@ -39,6 +39,7 @@
   %+  murn  ~(tap in scry-groups:grp)
   |=  rid=res
```

Comment:
This should probably be moved to a self-poke, so we can always scry, even if gall is molting